pinot-plugins/pinot-input-format/pinot-csv/src/test/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordReaderTest.java

```diff
@@ -112,20 +112,6 @@ protected void checkValue(RecordReader recordReader, List<Map<String, Object>> e
     Assert.assertFalse(recordReader.hasNext());
   }
 
-  @Test
-  public void testInvalidDelimiterInHeader() {
```

Comment:
Do we cover the case where `setDelimiter()` and actual delimiter for csv are different?

Reply:
When the set delimiter is different from the actual delimiter the entire line is read as a single column.